react/features/base/video-transform/functions.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import { toState } from '../redux';
+
+import type { Transform } from './constants';
+
+/**
+ * Returns a transform object currently stored for a participant's video.
+ *
+ * @param {Function|Object} stateful - The Redux state object or a function that
+ * returns the Redux state.
+ * @param {string} participantId - The participant ID.
+ * @returns {Transform}
```

Comment:
can the value returned here be undefined ? is the {Transform} type accurate here ?